sideloader/postinstall.sh

```diff
@@ -0,0 +1,9 @@
+pip="${VENV}/bin/pip"
+
+cd "${INSTALLDIR}/${REPO}/"
+
+$pip install -e "${INSTALLDIR}/${REPO}/"
+
+springboard="${VENV}/bin/springboard"
+
+$springboard bootstrap
```

Comment:
No, it leaves the `repos/` subfolder unless you pass `--clobber`. I do have a question about sideloader though... the `repos/` subfolder is created inside `springboard-iogt/`. Will sideloader delete `springboard-iogt/` on a re-deploy?